### PR TITLE
adding 10up package and WIP - just paragraph thus far

### PR DESCRIPTION
**Pull request by rayflores**
I added one package for functionality and registration of  ( make it easier ).  Also, this is WIP for just the paragraph at the moment... regard this as a POC.  This currently works on the `core/paragraph` block only.  

Rebuild by:
`npm install`
then
`npm run build`

### CODE REVIEW

**Review by iwatkinshpu — APPROVED** [review]

File: src/selector.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    />
+                </Panel>
+            </InspectorControls>
+        );
+    };
+    const generateClassName = (attributes) => {
+        let string = '';
+        if (attributes.hpuStyleClass) {
+            string += `${attributes.hpuStyleClass} `;
+        }
```
Comment (dswebsme) [inline src/selector.js:38]: Spacing.

File: src/selector.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        return string;
+    };
+    registerBlockExtension("core/paragraph", {
+        extensionName: 'hpu-blocks-core-selector',
+        attributes: additionalAttributes,
+        classNameGenerator: generateClassName,
+        Edit: BlockEdit,
+    });
```
Comment (dswebsme) [inline src/selector.js:46]: Spacing.

**Review by dswebsme — CHANGES_REQUESTED** [review]
@rayflores Most of what the original developer did with this code base was shared as a large completed code base with many WordPress coding standards excluded. As a development team, we've adopted those standards and try to ensure new code adheres as much as possible. I cited several examples here to get you started but please review the code I didn't comment on as well. I'm sure there will be unmentioned examples where WordPress coding standards for both PHP and JS could be applied. Thanks!

**Comment by rayflores** [comment]
@dswebsme I fixed the modified ( and surrounding ) formats and ran it through both the linter and code sniffer... All should be good.

File: hpu-blocks-core.php [hunk 3]
```diff
@@ -105,7 +105,30 @@ function hpu_blocks_core_register_block_scripts()
...
+	foreach ( $blocks as $block ) {
+		$src = plugin_dir_url( __FILE__ ) . '/build/style-index.css';
+		if ( is_admin() ) {
+			$src = plugin_dir_url( __FILE__ ) . '/build/index.css';
+		}
+		wp_enqueue_block_style(
+			$block,
+			array(
+				'handle' => 'hpu-blocks-core-editor-styles',
+				'src' => $src,
```
Comment (dswebsme) [inline hpu-blocks-core.php:126]: Consider:
```
'handle' => 'hpu-blocks-core-editor-styles',
'src'    => $src,
```
             ^ vertically aligned array values

**Review by dswebsme — APPROVED** [review]
@rayflores Approved. Just 1 item of note for future consideration. Looks great overall. Thanks!

**Pull request merged**